README.md

```diff
@@ -122,3 +157,38 @@ encode_number_precision
 This fork allows encoding of numbers with a `precision` up to 16 decimals (vs. 14 in mpx/lua-cjson).
 
 [Back to TOC](#table-of-contents)
+
+decode_array_with_array_mt
+--------------------------------
```

Comment:
Better remove the excessive dashes here.

Reply:
Oops, definitely an oversight, thanks for the catch!